### DISCUSSION TITLE
Option to hide played podcast episodes

### DISCUSSION THREAD

**Discussion by mariomaz87** [LeoKlaus/plappa#80] [Feature Requests]
Hi, I'm using Plappa only for podcasts using ABS and I'm loving it. I'm missing an option to hide played podcasts so that I can leave on the home screen only "to do" episodes. Thanks!

**Comment by LeoKlaus** [comment]
That's a pretty good idea, I'll make sure to add this!

**Reply by TobiasDax** [reply]
Hi, just found this discussion... 2years old, is there any progress or is this feature abandoned?
